langserver/hover.go

```diff
@@ -73,3 +106,43 @@ func packageStatementName(fset *token.FileSet, files []*ast.File, node *ast.Iden
 	}
 	return ""
 }
+
+// maybeAddComments appends the specified comments converted to HTML godoc form
+// as an text/html marked string to the specified contents slice, if the
+// comments string is not empty.
+func maybeAddComments(comments string, contents []lsp.MarkedString) []lsp.MarkedString {
+	if comments != "" {
+		var b bytes.Buffer
+		doc.ToHTML(&b, comments, nil)
+		return append(contents, lsp.MarkedString{
+			Language: "text/html",
+			Value:    b.String(),
+		})
+	}
+	return contents
```

Comment:
Did you write it this way in order to be more consistent with the 2 funcs below?

Instead of:

```Go
if comments == "" {
    return contents
}
... (less indent)
```